target/scripts/startup/setup.d/security/rspamd.sh

```diff
@@ -65,23 +66,59 @@ EOF
 # or checking for other anti-spam/anti-virus software.
 function __rspamd__run_early_setup_and_checks
 {
+  export RSPAMD_LOCAL_D='/etc/rspamd/local.d'
+  export RSPAMD_OVERRIDE_D='/etc/rspamd/override.d'
+  export RSPAMD_DMS_D='/tmp/docker-mailserver/rspamd'
```

Comment:
`export` needed?

```suggestion
  RSPAMD_LOCAL_D='/etc/rspamd/local.d'
  RSPAMD_OVERRIDE_D='/etc/rspamd/override.d'
  RSPAMD_DMS_D='/tmp/docker-mailserver/rspamd'
```

Reply:
Resolved in [dae7229](https://github.com/docker-mailserver/docker-mailserver/pull/3261/commits/dae722954c61a6c64084961d80e95c844559b50f)